kernels/portable/cpu/util/dtype_util.cpp

```diff
@@ -23,10 +23,14 @@ bool check_tensor_dtype(
       return executorch::runtime::tensor_is_realhbbf16_type(t);
     case SupportedTensorDtypes::REALHBF16:
       return executorch::runtime::tensor_is_realhbf16_type(t);
+    case SupportedTensorDtypes::REALH:
```

Comment:
what is the h in realh?

Reply:
yes, it matches the naming scheme in https://github.com/pytorch/executorch/blob/644b7ddf14180d97e348faa627f576e13d367d69/runtime/core/exec_aten/util/scalar_type_util.h#L1189